### ISSUE TITLE
Try if OptMeowt works on Android

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/gpc-optmeowt#426]
Just for fun.

[Prepare your Firefox desktop extension for the upcoming Android release](https://blog.mozilla.org/addons/2023/08/10/prepare-your-firefox-desktop-extension-for-the-upcoming-android-release/)

**Comment by SebastianZimmeck** [comment]
@OliverWang13, can you try?

**Comment by OliverWang13** [comment]
I do not have an Android but I can try to reach out to some people who do to test.

**Comment by SebastianZimmeck** [comment]
You can also try via an emulator in Android Studio (drag and drop the Firefox apk onto the screen).

**Comment by SebastianZimmeck** [comment]
Though, just installing from the Google Play Store should work just as well.

**Comment by OliverWang13** [comment]
I have an emulator up and running with Andriod Studio but have not been able to install optmeowt. I do not see it in the Google Play Store and the Firefox extension page does not allow for it to be installed. Attached is that view (add to firefox greyed out).


<img width="178" alt="Screen Shot 2023-09-10 at 4 21 14 PM" src="https://github.com/privacy-tech-lab/gpc-optmeowt/assets/82842273/b4f78491-b8f4-4e74-b4c7-a6ddf5f5538c">

**Comment by SebastianZimmeck** [comment]
@Jocelyn0830, if you have time, feel free to try as well. Does OptMeowt work on Firefox for Android?

**Comment by Jocelyn0830** [comment]
My finding is consistent with @OliverWang13's finding. 
I tested using the emulator on Android Studio.
I searched for OptMeowt in Firefox's add-on store and was able to find it. But the situation is exactly what Oliver found out. The extension page does not allow for it to be installed.

<img width="329" alt="Screen Shot 2023-09-25 at 14 18 42" src="https://github.com/privacy-tech-lab/gpc-optmeowt/assets/102534933/22aef8e5-7fbd-4ad7-ac7b-fb76a6c1bca9">
